### PR TITLE
fix --ww_ti_usb_reset

### PR DESCRIPTION
**Pull request by PieterGit**
@ceben80 noticed that the --ww_ti_usb_reset had gone broken on the dev branch. 

@ceben80 can you test this on my PieterGit:notify branch, or do you need it to be on dev branch to give it a try

### CODE REVIEW

File: bin/oref0-setup.sh [hunk 1]
```diff
@@ -141,7 +141,8 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
     read -p "What is your pump serial number (numbers only)? " -r
     serial=$REPLY
     echo "Ok, $serial it is."
-    read -p "What kind of CGM are you using? (e.g., G4-upload, G4-local-only, G5, MDT, xdrip?) Note: G4-local-only will NOT upload BGs from a plugged in receiver to Nightscout" -r
+    echo -e "What kind of CGM are you using?\nValid options:\nG4-upload: use Dexcom G4 receiver connected to USB and upload to nightscout\nG4-local-only: will NOT upload BGs from a plugged in receiver to Nightscout\nG5: Dexcom G5\nMDT: Medtronic CGM\nshareble: Bluetooth Dexcom G4 or G5 receiver\nxdrip: Requires xDrip Wireless Bridge (No Receiver)."
+    read -p "Please selection option for your CGM: " -r
```
Comment (scottleibrand) [inline bin/oref0-setup.sh]: Typo here.  Also would like to get input from folks like @kdisimone and @danamlewis on whether/how we want to change this text.

**Comment by PieterGit** [comment]
I also improved the formatting of the CGM questions a bit in this branch, notify stuff will be added later. Please don't merge to dev yet. I first want to check it on my own rig and want @ceben80 to give it a spin as well.

File: bin/oref0-setup.sh (hunk 1 above)
Comment (PieterGit) [inline bin/oref0-setup.sh]: @kdisimone and @danamlewis: please give advice on the best wording for each cgm option. I noticed shareble was missing and the formatting was very unclear, so I added newlines. Please make sure that CGM option + description stays within 80 characters limit.

**Comment by PieterGit** [comment]
Tested on my rig. Installs fine and looping is fine. @ceben80 (or some other PI3/TI USB users) are you able to give this a thumbs up as well?

**Comment by PieterGit** [comment]
@scottleibrand @ceben80: i have test this configuration for more almost two weeks. There is nothing I can think of that will break other use cases. Can you merge this to dev? I'll fix the typo and ask @Kdisimone and @danamlewis for further text improvements.

File: bin/oref0-setup.sh [hunk 2]
```diff
@@ -696,12 +710,15 @@ if [[ $ENABLE =~ microbolus ]]; then
...
-# Create ~/.profile so that openaps commands can be executed from the command line
-# as long as we still use enivorement variables it's easy that the openaps commands work from both crontab and from a common shell
-# TODO: remove API_SECRET and NIGHTSCOUT_HOST (see https://github.com/openaps/oref0/issues/299)
-echo Add NIGHTSCOUT_HOST and API_SECRET to $HOME/.profile
-(cat $HOME/.profile | grep -q "NIGHTSCOUT_HOST" || echo export NIGHTSCOUT_HOST="$NIGHTSCOUT_HOST") >> $HOME/.profile
-(cat $HOME/.profile | grep -q "API_SECRET" || echo export API_SECRET="`nightscout hash-api-secret $API_SECRET`") >> $HOME/.profile
+# Append NIGHTSCOUT_HOST and API_SECRET to $HOME/.bash_profile so that openaps commands can be executed from the command line
+# Once all crontab entries are called within a bash environment, then the NIGHTSCOUT_HOST and API_SECRET can be removed from the crontab 
+# TODO: remove API_SECRET and NIGHTSCOUT_HOST (see https://github.com/openaps/oref0/issues/299 )
+# Removing stuff from ~/.profile (old solution) is left as an exercise to the reader
```
Comment (scottleibrand) [inline bin/oref0-setup.sh]: Which will take precedence, the ~/.profile or the ~/.bash_profile entries?  Either we need to choose to just leave the old cruft there (if it's harmless) or we need to remove it.  We can't expect anyone to read this code and go clean things up.

File: bin/oref0-setup.sh [hunk 3]
```diff
@@ -696,12 +710,15 @@ if [[ $ENABLE =~ microbolus ]]; then
...
+echo "#Added NIGHTSCOUT_HOST and API_SECRET to $HOME/.bash_profile at `date`" | tee -a $HOME/.bash_profile
+echo NIGHTSCOUT_HOST="$NIGHTSCOUT_HOST" >> $HOME/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-setup.sh]: These will append every time oref0-setup is run, filling up .bash_profile with old / redundant entries.  Why did you remove the `grep -q keyword || ` stuff?

File: bin/oref0-setup.sh (hunk 2 above)
Comment (PieterGit) [inline bin/oref0-setup.sh]: fixed on https://github.com/openaps/oref0/pull/470

File: bin/oref0-setup.sh (hunk 3 above)
Comment (PieterGit) [inline bin/oref0-setup.sh]: fixed on https://github.com/openaps/oref0/pull/470

**Comment by scottleibrand** [comment]
I believe this has now been merged via #470

**Comment by PieterGit** [comment]
Yes, has been merged together with 470. I tested it just before my new explorer boards arrived on a rpi3. Closing issue
